### ISSUE TITLE
Connecting to WiFi Network the First Time Causes UI to Freeze

### ISSUE THREAD

**Issue by rstrube** [351ELEC/351ELEC#436]
### Describe the bug
Connecting to WiFi network for the first time causes the UI to freeze.  You must toggle the power button on the RG351P to restore UI interaction.

### To Reproduce
1. Go to Network Settings
2. Setup your WiFi SSID and WiFi Key
3. After entering the WiFi Key the system will attempt to connect to the WiFi network.
4. The UI will freeze
5. Toggling the power button will cause the UI to un-freeze and you'll be presented with a dialog box telling you the WiFi has been connected.  UI interaction is restored.

### Expected behavior
The UI shouldn't freeze after entering WiFi credentials.  You shouldn't have to toggle the power button for the UI to become responsive again.

### Device (please complete the following information):
 - RG351P device
 - Issue present on 20210510 Reborn Release
 - Issue present building from `main` as of commit af0da95ba61896c20b969267cdec3e6671bd57d7

**Comment by baskapteijn** [comment]
I want to add that this also happens with:

- RG351V
- On 20210510 Reborn Release

**Comment by szalik-rg351** [comment]
truth with wifi freeze i have same thing

**Comment by baskapteijn** [comment]
This also happens with a fresh install of release `351ELEC-RG351V.aarch64-20210603.img.gz`, This may be stating the obvious but I think it's relevant info nevertheless.

**Comment by rstrube** [comment]
I just conducted a fresh install of `351ELEC-RG351M.aarch64-20210603` (hotfix 3) and the UI freeze felt more reasonable. The dialog box that pops up telling the user that you've successfully connected appeared after 4-5 seconds.  During that time the UI was "frozen" but I didn't have to toggle the power button like I had to in the past.  Perhaps this is something that should be addressed upstream with Emulation Station? (e.g. adding some sort of visual feedback that an action is occurring in the background).
